src/pages/NewChatConfirmPage.tsx

```diff
@@ -151,11 +163,12 @@ function NewChatConfirmPage({newGroupDraft, allPersonalDetails}: NewChatConfirmP
                     }}
                     size={CONST.AVATAR_SIZE.XLARGE}
                     avatarStyle={styles.avatarXLarge}
-                    shouldDisableViewPhoto
+                    shouldDisableViewPhoto={!avatarFile}
```

Comment:
The `shouldDisableViewPhoto` prop should be removed and we should relay only on `isUsingDefaultAvatar` to disable the view photo option.